tagstudio/src/core/constants.py

```diff
@@ -7,6 +7,11 @@
 COLLAGE_FOLDER_NAME: str = "collages"
 LIBRARY_FILENAME: str = "ts_library.json"
 
+FONT_SAMPLE_TEXT: str = (
+    """ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!?@$%(){}[]"""
+)
+FONT_SAMPLE_SIZES: list[int] = [12, 15, 20]
```

Comment:
Slight adjustment to font sample sizes to show more difference between them
```suggestion
FONT_SAMPLE_SIZES: list[int] = [10, 15, 20]
```